Gemfile.lock

```diff
@@ -57,10 +57,10 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 0aea5769bad46b9da4e60f8d40c613ad7a1c6bad
+  revision: d8a039099a64a198c45280bf4ca83cabd2a2968d
```

Comment:
This hash will be updated to the one generated when [this pr](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/535) is merged into master.